<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find_Widget</class>
 <widget class="QWidget" name="Find_Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>178</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <widget class="QOpenGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>280</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>93</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Button_CheckNext">
      <property name="text">
       <string>查找下一个</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FindButton_Cancle">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="FindCheckBox_Isupper">
      <property name="text">
       <string>区分大小写</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBox_Loop">
      <property name="text">
       <string>循环</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="title">
    <string>方向</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="RadioUp">
      <property name="text">
       <string>向上</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="RadioDown">
      <property name="text">
       <string>向下</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Labe_Context">
      <property name="text">
       <string>查找内容:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
